src/components/IconCustomizer.svelte

```
@@ -1,49 +1,30 @@
-					_.flow(uiStateEx.stylesFromUserIcon, uiState.upsertStyles)(userIcon);
-
-					if (isContentTypeSvg(contentType)) {
-						uiState.selectSvgIcon(iconContent);
-					} else {
-						_.flow(IconService.mkIconUrl, uiState.selectImageIcon)(userIcon.originalIconId);
-					}
-				} catch (error: any) {
-					throw new Error('Error fetching user icon', error);
-				}
-			})();
+		if ($customizedIcon.userIconId && $customizedIcon.userIconId !== UUID.empty) {
```

ESLint

[error] Properties of objects and arrays are not reactive unless in runes mode. Changes to this property will not cause the reactive statement to update(reactive_declaration_non_reactive_property)

(svelte/valid-compile)